dev/pragma.h

```diff
@@ -117,17 +117,18 @@ namespace sqlite_orm {
                 return this->get_pragma<std::vector<std::string>>(oss.str());
             }
 
-            std::vector<sqlite_orm::table_info> table_info(const std::string& tableName) const {
+            // will include generated columns in response as opposed to table_info
+            std::vector<sqlite_orm::table_xinfo> table_xinfo(const std::string& tableName) const {
```

Comment:
Namespace qualifier can be omitted.

Reply:
not really, colides with method name table_xinfo